### DISCUSSION TITLE
Enable ipv6 in coolify network

### DISCUSSION THREAD

**Discussion by Vahor** [coollabsio/coolify#4048] [Improvement Requests]
### Description

I need to access the ipv6 of an app running in a container. 
But the network does not seem to be able to access those ips.

I can delete the `coolify` network and recreate it with [ipv6 support](https://docs.docker.com/engine/daemon/ipv6/) but as it's also managed by coolify it seems hacky and doing so on every server does not seem nice.
(That's what I'm currently doing)

[Doc](https://coolify.io/docs/knowledge-base/docker/custom-commands) is not very clear about what the flag does, but I also tried with `--ip6`, it does not change anything

### Minimal Reproduction (if possible, example repository)

Inside a container using the coolify network
or `docker run --rm -it --network coolify ubuntu`
```
apt update && apt install curl -y
curl https://ipv6.icanhazip.com/
```

You'll see an error
```
curl: (7) Failed to connect to ipv6.icanhazip.com port 443 after 3 ms: Couldn't connect to server
```
(works for ipv4)

Note: or simpler by installing `net-tools` and running `ifconfig`, you'll see that with coolify network you don't have the inet6, but without the coolify network an address is here.

For comparison, you can run a container using the host network `docker run --rm -it ubuntu`. Same commands, here it woks.

### Version

v4.0.0-beta.323

### Cloud?

- [ ] Yes
- [X] No

**Comment by Vahor** [comment]
https://github.com/coollabsio/coolify/issues/2495 can also be a solution, adding a custom network with ipv6 and using this one will solve my problem.
Edit: we'd still have to enable the ipv6 support on docker

**Comment by Jackman3005** [comment]
Hey, we're running into this issue when trying to connect to a supabase DB which only support ipv6 for direct connections. @Vahor I'm looking into doing what you're doing, but I'm concerned I will recreate it with missing configuration. How did you recreate it exactly?

**Comment by Cybertronus** [comment]
Would also love IPv6 support for Coolify networks. Same issue as @Jackman3005. I don't want to manually recreate a Coolify-managed resource, especially with no way to configure it in the UI itself

**Comment by ARHAEEM** [comment]
Just got this issue... Living with an IPv6 host is the worst thing I’ve ever done.

**Comment by hahagu** [comment]
Are there any plans to resolve this in the near future?

**Comment by hahagu** [comment]
A workaround for this is to enable ipv6 on docker by adding
```
{
  "ip6tables": true,
  "experimental": true
}
```
to `/etc/docker/daemon.json`
and checking `Advanced >  Connect To Predefined Network`.
IPV6 connectivity will work for the container then.

**Comment by A1bi** [comment]
Coolify also should host their images (e.g. `coolify-helper`) elsewhere since GitHub (and its container registry `ghcr.io`) famously is still IPv4-only in 2026.

**Reply by djsisson** [reply]
images are also on docker hub

there is an env you can set ```REGISTRY_URL``` to change the default registry from ghcr.io, this can be set at ```/data/coolify/source/.env```

**Reply by A1bi** [reply]
Oh great, thank you!
